### ISSUE TITLE
docker image for v3.8.6 not starting up

### ISSUE THREAD

**Issue by oliveregger** [ahdis/matchbox#258]
- txCache was not set up properly
- applicaton.yaml needs to adapted if you are using the CDA Logical model

from

```yaml
      cda:
        name: hl7.cda.uv.core
        version: 2.0.0-sd-202312-matchbox-patch
        url: classpath:/hl7.cda.uv.core#2.0.0-sd-202312-matchbox-patch.tgz
```

to

```yaml
      cda:
        name: hl7.cda.uv.core
        version: 2.0.0-sd-202406-matchbox-patch
        url: classpath:/hl7.cda.uv.core#2.0.0-sd-202406-matchbox-patch.tgz
```

**Comment by oliveregger** [comment]
fixed with commit https://github.com/ahdis/matchbox/commit/dbf86be7f59c8caea5b956f388542aec9d713f50

**Comment by oliveregger** [comment]
fixed in version 3.8.8
